test/com/facebook/buck/android/testdata/android_project/kotlin/com/sample/lib/BUCK.fixture

```diff
@@ -0,0 +1,35 @@
+android_library(
+  name = 'lib',
+  srcs = glob(['*.kt']),
+  language = 'KOTLIN',
```

Comment:
The only thing `java_library()` was allowed to receive in its `srcs` in the past were files that ended in `.java` or `.src.zip`. Ignoring `.src.zip` for the moment, `language` seems like something that the `Description` could figure out, so I don't see why this is a big deal. I would rather ban the `language` param quickly before people start relying on it.

Also, I don't see `language` documented on https://buckbuild.com/rule/android_library.html =\